app/assets/javascripts/folio/api.js

```diff
@@ -59,6 +59,7 @@ const makeCheckResponse = (asHtml) => (response) => {
         return Promise.resolve(json)
       }
 
+      window.FolioConsole.Flash.flashMessageFromApiErrors(json)
```

Comment:
ne, tady to byt nemuze, to by se delalo ve vsech requestech i na frontendu atd.

Reply:
musi to byt primo u te aasm komponenty - grepni si v projektu `flashMessageFromApiErrors`, abys videl, jak se to pouziva